pyroma/__init__.py

```diff
@@ -10,7 +10,8 @@
 
 def zester(data):
     main_files = os.listdir(data["workingdir"])
-    if "setup.py" not in main_files and "setup.cfg" not in main_files:
+    # There has to be at least one standard Python config file
+    if "setup.py" not in main_files and "setup.cfg" not in main_files and "pyproject.toml" not in main_files:
```

Comment:
I was thinking about it, but I thought it was ugly, and since I'm a moron and I can't under the code I wrote three months ago unless it's extremely clear, I decided against that. But now I just realized I can just make a not and an intersection, which I find completely readable. :-)

Reply:
Yeah, that was my one concern was suggesting this, though ameliorated by your helpful comment above it. I also considered suggesting the not-intersection approach, i.e. 

```python
if not {"setup.py", "setup.cfg", "pyproject.toml"}.intersection(main_files):
```

but I wasn't sure myself that was any clearer, heh.